rebar.config

```diff
@@ -18,10 +18,6 @@
 
 {escript_name, rebar3}.
 {escript_emu_args, "%%! +sbtu +A0\n"}.
-%% escript_incl_extra is for internal rebar-private use only.
-%% Do not use outside rebar. Config interface is not stable.
-{escript_incl_extra, [{"relx/priv/templates/*", "_build/default/lib/"},
-                      {"rebar/priv/templates/*", "_build/default/lib/"}]}.
```

Comment:
true, this depends on the outcome of the next comment though, if we build with a nightly profile relx will end up in _build/nightly/lib and not in _build/default/lib, maybe a {{profile}} var is needed